### PR TITLE
FE-2309 stop additional onclick submitting form

### PR DESCRIPTION
**Pull request by DipperTheDan**
# Description
Add to type property to button component to prevent additional button submitting form.

### CODE REVIEW

File: src/__deprecated__/components/form/form-button/form-button.component.js [hunk 1]
```diff
@@ -16,6 +16,7 @@ const propsForButton = (props) => {
   return ({
     save: {
       as: 'primary',
+      type: undefined,
```
Comment (jamime) [inline src/__deprecated__/components/form/form-button/form-button.component.js]:
```suggestion
      type: 'submit',
```

Comment (jamime) [reply src/__deprecated__/components/form/form-button/form-button.component.js]: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/submit

File: change_log/next/FE-2309-fix_child_button_submitting_form.yml [hunk 2]
```diff
@@ -0,0 +1 @@
+Bug Fixes: "Adds type property to button component to stop form submission on click. Removes type property from save button in form. (component: Button, FormButton)"
```
Comment (grabkowski) [inline change_log/next/FE-2309-fix_child_button_submitting_form.yml]: I think that could be split into two separate entries, e.g.:
```
Bug Fixes: "Adds type property to button component to stop form submission on click.  (component: Button)"
Bug Fixes: "Removes type property from save button in form. (component: FormButton)"
```

File: src/__deprecated__/components/form/form-button/form-button.component.js (hunk 1 above)
Comment (DipperTheDan) [reply src/__deprecated__/components/form/form-button/form-button.component.js]: Made the change locally as I had to update the snapshot. Thank you.

**Pull request merged**